examples/game_of_life/_helpers.py

```diff
@@ -0,0 +1,302 @@
+"""Helper classes and functions for MeasurementLink examples."""
+
+import logging
+import pathlib
+import types
+from typing import (
+    Any,
+    Callable,
+    List,
+    NamedTuple,
+    Optional,
+    Tuple,
+    TypeVar,
+    Union,
+)
+
+import click
+import grpc
+
+import ni_measurementlink_service as nims
+from ni_measurementlink_service import session_management
+from ni_measurementlink_service._internal.discovery_client import DiscoveryClient
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.pinmap.v1 import (
+    pin_map_service_pb2,
+    pin_map_service_pb2_grpc,
+)
+from ni_measurementlink_service.measurement.service import (
+    GrpcChannelPool,
+    MeasurementService,
+)
+
+
+class ServiceOptions(NamedTuple):
+    """Service options specified on the command line."""
+
+    use_grpc_device: bool = False
+    grpc_device_address: str = ""
+
+    use_simulation: bool = False
+
+
+def get_service_options(**kwargs) -> ServiceOptions:
+    """Get service options from keyword arguments."""
+    return ServiceOptions(
+        use_grpc_device=kwargs.get("use_grpc_device", False),
+        grpc_device_address=kwargs.get("grpc_device_address", ""),
+        use_simulation=kwargs.get("use_simulation", False),
+    )
+
+
+T = TypeVar("T")
```

Comment:
I'm not seeing T used anywhere. Can we remove it?

Reply:
I also don't see it used. We have 'vendored' this file in all our examples and the template, so we have been trying to keep all of them identical. @bkeryan can we remove this from all of the _helpers.py example files and from the template? If so, I'll plan to do this in a separate PR.

Reply:
`T` was used by `str_to_enum`. When we added support for real enums, we removed `str_to_enum` but we overlooked `T`.

Reply:
Removed. Also from the rest of the _helpers.py in #363